pkg/network/usm/monitor_windows.go

```diff
@@ -55,7 +55,7 @@ func NewWindowsMonitor(c *config.Config, dh driver.Handle) (Monitor, error) {
 		di:         di,
 		hei:        hei,
 		telemetry:  telemetry,
-		statkeeper: http.NewStatkeeper(c, telemetry),
+		statkeeper: http.NewStatkeeper(c, telemetry, http.NewIncompleteBuffer(c, telemetry)),
```

Comment:
That's the current windows implementation, but it can work for http2 in the middle ground
given the fact we're going to implement a new incomplete buffer for http2, does it make any sense to still have noopIncompleteBuffer?

Reply:
ah, sorry. I didn't realize Windows had such a thing already.